google-cloud-nio/src/main/java/com/google/cloud/storage/contrib/nio/CloudStorageFileSystemProvider.java

```diff
@@ -87,6 +87,8 @@
 @ThreadSafe
 public final class CloudStorageFileSystemProvider extends FileSystemProvider {
 
+  public static final String BUCKET_IS_REQUESTER_PAYS_ERROR_MESSAGE =
+      "Bucket is a requester pays bucket but no user project provided";
```

Comment:
@lbergelson have you considered a simpler match string that may have a better chance of not breaking in the future?

For example, it looks like @breilly2 and @mcovarr updated cromwell to use "requester pays bucket but no user project": 
https://github.com/broadinstitute/cromwell/pull/6689

Meanwhile terra-ui searches for the even more basic string "requester pays", and didn't even need to be updated with the last server-side change: https://github.com/DataBiosphere/terra-ui/blob/e34bdf0b81d7d2750787108ac06238787a363726/src/libs/ajax.js#L76